Sprint-1/explore/chrome.md

```diff
@@ -10,9 +10,9 @@ Let's try an example.
 In the Chrome console,
 invoke the function `alert` with an input string of `"Hello world!"`;
 
-What effect does calling the `alert` function have?
+What effect does calling the `alert` function have? I get a pop-up window which prints my message.
 
 Now try invoking the function `prompt` with a string input of `"What is your name?"` - store the return value of your call to `prompt` in an variable called `myName`.
 
-What effect does calling the `prompt` function have?
-What is the return value of `prompt`?
+What effect does calling the `prompt` function have? I get a pop-up notification asking the question and providing a box to get the input.
+What is the return value of `prompt`? it outputs the input of that question.
```

Comment:
What would prompt() return if a user doesn't enter any value and hit 'OK'?
What would prompt() return if a user clicks 'Cancel' instead of 'OK'?